login-workflow/src/new-architecture/screens/AccountDetailsScreen/AccountDetailsScreen.tsx

```diff
@@ -20,7 +20,7 @@ export const AccountDetailsScreen: React.FC<AccountDetailsScreenProps> = (props)
             await actions().setAccountDetails({ firstName, lastName });
             void nextScreen({
                 screenId: 'AccountDetails',
-                values: { firstName, lastName },
+                values: { firstName: firstName, lastName: lastName },
```

Comment:
You should be able to undo this change. No need to explicitly name them like this.